releases/eol-policy.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Product Support Lifecycle and EOL Policy
+summary: 
+toc: true
+---
+
+## Product Release Numbering
+
+CockroachDB uses a three-place numbering scheme to designate [released versions](../releases/) of software. The format is YY.R.PP, where YY indicates the year, R indicates release with “1” for Spring and “2” for Fall, and PP indicates the patch release version. An example would be Version 20.1.1 (abbreviated v20.1.1).
+
+{{site.data.alerts.callout_info}}
+This versioning scheme began with v19.1. Prior releases use a different versioning scheme.
+{{site.data.alerts.end}}
+
+- A major release is produced twice a year indicating major enhancements to product functionality. A change in the YY.R component denotes a major release.
+- A patch (or maintenance) release is produced to roll out critical bug and security fixes. A change in the PP numbering denotes a patch release.
+
+## Support Cycle
+
+When a major release is made available, Cockroach Labs will continue to provide **Maintenance Support** for the prior major release for at least 365 days from the declared production date of the new major release. After this period, Cockroach Labs will provide **Assistance Support** for an additional 180 days at minimum.
```

Comment:
I find this confusing, and feel it doesn't align with the calendar below?

Also, how does this interact with our (IMO simpler) policy of supporting the latest stable release and two releases prior?